quantum-template.tex

```diff
@@ -171,6 +169,11 @@ \section{Unpublished and accepted works}
 It can be used for works not yet accepted in Quantum or not even intended for submission to Quantum.
 On the contrary, by giving the \texttt{accepted=YYYY-MM-DD} option, with \texttt{YYYY-MM-DD} the acceptance date, the note ``Accepted in Quantum YYYY-MM-DD, click title to verify'' can be added to the bottom of each page to clearly mark works that have been accepted in Quantum. 
 
+\section{Plots}\label{sec:plots}
+Quantum provides a Jupyter notebook based on the widely used library \texttt{matplotlib} that greatly simplifies the creation of plots that integrate seamlessly with the \texttt{quantumarticle} class. This is intended as a service to the authors and is \textit{not} mandatory. The notebook can be found in the \href{https://github.com/quantum-journal/quantum-journal}{GitHub repository}. The Python notebook relies. It only requires the author to specify font size and paper format that were passed as options to \texttt{quantumarticle}.
+
+We strongly encourage authors to use a lossless vector format for their submissions, \textit{e.g.}\ Pdf or Pgf. 
```

Comment:
```suggestion
We strongly encourage authors to use the PDF format for their submissions.
```

Reply:
We want authors to use pdflatex on the arXiv, because we need the arXiv to produce a pdf.

Reply:
I was referring to the format of the figures, but I think just staying with PDF is clearer. The quality of PDF is the same as PGF and you can at least look at it without compiling it first.